### PR TITLE
Add Shared Schemes for EarlGrey.xcodeproj and FunctionalTests.xcodeproj

### PR DESCRIPTION
**Pull request by tirodkar**

### CODE REVIEW

File: EarlGrey.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (khandpur) [inline EarlGrey.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: is this file required?

File: Tests/FunctionalTests/FunctionalTests.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (khandpur) [inline Tests/FunctionalTests/FunctionalTests.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: is this file required?

File: EarlGrey.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 1 above)
Comment (tirodkar) [reply EarlGrey.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Fixed.

File: Tests/FunctionalTests/FunctionalTests.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 2 above)
Comment (tirodkar) [reply Tests/FunctionalTests/FunctionalTests.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Fixed.
